pages/[[server]]/list/[list]/index.vue

```diff
@@ -17,15 +17,15 @@ const tabs = computed<CommonRouteTabOption[]>(() => [
       name: 'list',
       params: { server: server.value, list: list.value },
     },
-    display: t('tab.posts'),
+    display: isHydrated.value ? t('tab.posts') : '',
```

Comment:
Could this one be removed too? And the one below?

Reply:
done